networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/gossip/AttestationGossipManager.java

```diff
@@ -63,7 +63,7 @@ public void onNewAttestation(final ValidateableAttestation validateableAttestati
             },
             error -> {
               LOG.log(
-                  error instanceof NoPeersForOutboundMessageException ? Level.DEBUG : Level.WARN,
+                  error instanceof NoPeersForOutboundMessageException ? Level.DEBUG : Level.ERROR,
```

Comment:
🤦‍♂️

Reply:
> For `HistoricalBlockSyncService` we are doing an initial `fetchBlocks()` already, it's just chain after initialise:
> 
> ```
> 
>   @Override
>   protected SafeFuture<?> doStart() {
>     LOG.debug("Start {}", getClass().getSimpleName());
>     return initialize().thenRun(this::fetchBlocks);
>   }
> ```
> 
> so could remove the `thenRun` from that if we know that the sync state subscription will always call us back.

It's a bit more complicated than that. If I do the change there is a side effect when we start the service and we are already in sync that cause us to not unsubscribe since we call stop() before actually set the subscriptionId in `syncStateSubscription` (which remains -1).

Reply:
> This change is from the other PR and probably shouldn't be in here. :)

I leave it here since ERROR is what it should be.

Reply:
> It's a bit more complicated than that. If I do the change there is a side effect when we start the service and we are already in sync that cause us to not unsubscribe since we call stop() before actually set the subscriptionId in syncStateSubscription (which remains -1).

Ah subtle - good spot.